inductiva/__init__.py

```diff
@@ -14,7 +14,7 @@
 from . import generative
 from . import molecules
 from . import resources
-# from . import structures
```

Comment:
I think that this had been commented to make sure there are no problems with imports of external libraries. Can you please check with @luispcunha and @IvanPombo that this won't break our "import inductiva"?

Reply:
This is due to the gmsh not being anymore on the Client. I will check on this and you can notify me if it works on your side. We may need to follow the approach of this PR #538.

Reply:
@hpenedones, I uncommented it because I thought that this problem had been resolved
@IvanPombo, thanks. I will now verify if everything is functioning correctly

Reply:
I believe it may be working for you, but not for other users. So I think we should try to pass the gmsh dependencies to the backend - specially because this is a scenario.

Reply:
Should I uncomment it now and then pass the Gmsh to the backend since this will be my next task?

Reply:
Just uncomment, only if you are not using the gmsh anymore. Otherwise, and if you want to be able to merge keep it commented while you work. We will be deploying a new version of the package today and we do not want that to break it apart. Thanks :)